### DISCUSSION TITLE
VPS machine specs and how much I can go with this single server?

### DISCUSSION THREAD

**Discussion by djalmaaraujo** [socketio/socket.io#4959] [Q&A]
Hello! I am building a text chat only, with multiple 2-person rooms.

I deployed in a VPS with 6GB of RAM, 1.5TB bandwidth, 4 vCPUs.

I would like to ask how many users do you believe this server could really support at the same time without too much of an issue?

Based on the Socket.io memory usage looks like could go easy with 3k simultaneous, but I would like to ask for people with a better "live" experience.

Thank you!

**Comment by darrachequesne** [comment]
Hi! The memory and CPU usage will vary a lot depending on your own usage (usage of rooms, number of message per seconds, average size of each message, ...).

Without any additional information, I'd say your VPS should easily handle 3000 simultaneous users, but I strongly advise to do your own benchmarks.
